queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py

```diff
@@ -107,7 +107,6 @@ def correct_business_data(business: Business,  # pylint: disable=too-many-locals
     # update resolution, if any
     with suppress(IndexError, KeyError, TypeError):
         resolution = dpath.util.get(correction_filing, '/correction/resolution')
-        filings.update_filing_json(correction_filing_rec, resolution)
```

Comment:
What is this change?

Reply:
Redundant. Getting resolution from `/correction/resolution` and setting to the same property